app/assets/v2/js/pages/fulfill_bounty/index.js

```diff
@@ -1,4 +1,20 @@
 /* eslint-disable no-console */
+needWalletConnection();
+
+const fetchFromWeb3Wallet = () => {
+  if (!provider) {
+    onConnect();
+  }
+  $('#payoutAddress').val(selectedAccount);
+  $('#payoutAddress').attr('readonly', true);
+}
+
+window.addEventListener('dataWalletReady', function(e) {
+  if (is_bounties_network) {
+    fetchFromWeb3Wallet();
+  }
+}, false);
```

Comment:
^ do we wanna just move this into ETH.js + how would this change once we retire std bounties ?
something like 

```
if (is_bounties_network || web3_modal) {
...
}
```